lib/stampable.rb

```diff
@@ -46,6 +46,11 @@ def self.included(base) #:nodoc:
           # Defaults to :deleter_id when compatibility mode is off
           # Defaults to :deleted_by when compatibility mode is on
           class_attribute  :deleter_attribute
+
+          # Define if the belongs_to association should be optional or not
+          # Defaults to true to mantein the old behavior
```

Comment:
Actually the old behaviour of Rails prior to 5.1 is `optional: true`

We should reflect that. Also there is a little typo in the comment ("maintein" vs "maintain")

Reply:
and if we let it change to reflect the configuration of the Rails version?
Rails >=5.1 default => false
Rails <5.1 default => true